spec/classes/backend_spec.rb

```diff
@@ -7,7 +7,9 @@
       describe 'with default values for all parameters' do
         it { should compile.with_all_deps }
 
-        it { should contain_class('sensu::backend')}
+        it { should contain_class('sensu::backend') }
+        it { should contain_class('sensu') }
+        it { should_not contain_class('sensu::agent') }
```

Comment:
Would not expect agent class to be here.

Reply:
The test is using `should_not` so testing that `sensu::agent` isn't in catalog for just backend.